src/python/install.sh

```diff
@@ -26,7 +26,7 @@ CONFIGURE_JUPYTERLAB_ALLOW_ORIGIN="${CONFIGUREJUPYTERLABALLOWORIGIN:-""}"
 # alongside PYTHON_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv" "pytest")
+DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv" "pytest" "ruff")
```

Comment:
@bamurtaugh any thoughts on adding this?

Reply:
Thanks for opening @basnijholt! I'd love to get @brettcannon's and @luabud's expertise here.

Reply:
Ruff is definitely a reasonable thing to include, especially based on what's in that list already and overall popularity (e.g., I would argue Ruff is more popular than a 1/3 of that list already).

Reply:
Great, thanks for your insights. @bamurtaugh let's add ruff? I can help review this PR.

Reply:
Sounds good to me! Thanks @brettcannon and @samruddhikhandale

Reply:
comment